.sops.yaml

```diff
@@ -9,6 +9,7 @@ keys:
   - &ryantm age1d87z3zqlv6ullnzyng8l722xzxwqr677csacf3zf3l28dau7avfs6pc7ay
   - &zimbatm age1jrh8yyq3swjru09s75s4mspu0mphh7h6z54z946raa9wx3pcdegq0x8t4h
   - &zowoq age1m7xhem3qll35d539f364pm6txexvnp6k0tk34d8jxu4ry3pptv7smm0k5n
+  - &adisbladis 00244EF5295026AA323A4BDB110BFAD44C6249B7
```

Comment:
This isn't a valid key?

Reply:
Ah, gpg. We got rid of that, could you please use an age key instead?

Reply:
I find using age keys more inconvenient as I don't already have that integrated into my key management workflow.
I've generated an age key but I think it's a pretty massive PITA to manage more keys.

Reply:
To elaborate a bit on as to _why_ it is a PITA:
I have a strict policy not to have keys on disk but only in crypto hardware such as a Yubikey.

My GPG was generated in an air gapped environment and I have physical backups residing in other Yubikey's & one that can be recovered by assembling all the parts (https://en.wikipedia.org/wiki/Shamir%27s_secret_sharing) which are spread in multiple physical locations.
Setting up this kind of key management is not exactly trivial and takes a lot of effort.
Since I went through this effort in 2017 I have not had to do it again.

This key was now generated ad-hoc and I'm going to have to regenerate it again and give it the full treatment.